buildpipeline/DotNet-Trusted-Publish.json

```diff
@@ -594,11 +594,11 @@
       "allowOverride": true
     },
     "AzureContainerPackageGlob": {
-      "value": "pkg\\*.nupkg",
+      "value": "pkg\\*\\*\\*.nupkg",
       "allowOverride": true
     },
     "AzureContainerSymbolPackageGlob": {
-      "value": "symbolpkg\\*.nupkg",
```

Comment:
Is this the only glob path that needed to be changed?

Reply:
This is the same as what is in publish-symbols.json. I tried ** there & it didn't work for some reason